integration/raft/channel_participation_test.go

```diff
@@ -60,29 +63,34 @@ var _ = Describe("ChannelParticipation", func() {
 		os.RemoveAll(testDir)
 	})
 
-	Describe("basic etcdraft network without a system channel", func() {
+	Describe("three node etcdraft network without a system channel", func() {
+		startOrderer := func(o *nwo.Orderer) {
+			ordererRunner := network.OrdererRunner(o)
+			ordererProcess := ifrit.Invoke(ordererRunner)
+			Eventually(ordererProcess.Ready, network.EventuallyTimeout).Should(BeClosed())
```

Comment:
Ready is a method that returns a channel. You should actually be calling the method here.

I just fixed a problem like this in the health check tests where the test passed the method to _get_ the channel instead of the result of getting the channel to the matcher.

Reply:
Done